docs-gen/content/rule_set/data_entry/allowed.md

```diff
@@ -28,10 +29,43 @@ specified in this list.
 The `datatype` specifier gives the type of the individual elements of the `allowed`
 list.
 
+## Recommendation on String values
+
 For string values used for `default` and `allowed` statements it is recommended to start with `A-Z`
 and then use only `A-Z`, `0-9` and underscore (`_`).
 It is recommended to use single quotes (`'`) as tooling otherwise might handle literals like `OFF` as boolean values with unexpected result.
 It is recommended not to specify a dedicated value corresponding to "unknown" or "undefined" unless there is a relevant use-case for that particular signal.
 The background is that a signal with an array of allowed values shall be handled just as any other signal.
 If e.g. the value of current speed or vehicle weight is unknown, then the vehicle shall not publish the corresponding signal.
 Similarly, for the example above, if the steering wheel position is unknown then `SteeringWheel.Position`shall not be published.
+
+## Allowed values for array types
+
+The `allowed` keyword can be used also for signals of array type. In that case `allowed` specifies valid entries in the array.
```

Comment:
can be used also -> can also be used

in that case -> in that case,

specifies valid entries in the array -> specifies the only valid values for array elements

Reply:
Thanks for the comments, will update the PR.